### PR TITLE
Better "tracked" icon, pro player badge, fixes

### LINKED ISSUE #161: display "inactive" icon
* Inactive: Currently the behavior is to display the "tracked" icon if tracked and nothing if the user is not tracked.  I think it's beneficial to display for example a "Zzz" icon to indicate the user is inactive.  That way the user can more quickly figure out why their matches aren't being parsed.

* Currently no icon is displayed for 0 cheese.  I recently added this in the current UI to aid spacing/layout issues (easier to debug if it shows up on all pages).

### LINKED ISSUE #174: No hero (unplayed match)
/players/26316691
/matches/2661560283

image

### PR DESCRIPTION
**Pull request by blukai**
fixes #161 
fixes #174 
@howardchung, Should I add some explanation into untracked tooltip?

### CODE REVIEW

File: components/Player/PlayerBadges.jsx [hunk 1]
```diff
@@ -1,66 +1,50 @@
...
-        {Math.round(new Date().getTime() / 1000.0) >= Number(tracked) ? null :
-          <div>
-            <div data-tip data-for="tracked">
-              <ActionVisibility className={styles.icon} />
-            </div>
+        {Math.round(new Date().getTime() / 1000.0) >= Number(tracked) ?
+          <div data-tip data-for="untracked" className={styles.iconButton}>
+            <IconEyeInactive className={styles.icon} style={{ height: 22, fill: 'darkgray' }} />
+            <ReactTooltip id="untracked" place="top" type="light" effect="float">
+              This user is untracked.
```
Comment (howardchung) [inline components/Player/PlayerBadges.jsx]: This user is inactive, and replays of new matches will not be automatically parsed.

File: components/Player/PlayerBadges.jsx [hunk 2]
```diff
@@ -1,66 +1,50 @@
...
+            </ReactTooltip>
+          </div>
+        :
+          <div data-tip data-for="tracked" className={styles.iconButton}>
+            <IconEye className={styles.icon} style={{ height: 22 }} />
             <ReactTooltip id="tracked" place="top" type="light" effect="float">
               This user is tracked.
```
Comment (howardchung) [inline components/Player/PlayerBadges.jsx]: to match the other:

This user is tracked, and replays of new matches will be automatically parsed.

File: components/Player/PlayerBadges.jsx [hunk 3]
```diff
@@ -1,66 +1,50 @@
...
-                  height: 18,
-                  fill: 'white',
-                  stroke: 'white',
-                  strokeWidth: 3,
-                  paddingTop: 3,
-                  marginLeft: 4,
-                }}
-              />
-            </div>
+        {cheese !== 0 &&
```
Comment (howardchung) [inline components/Player/PlayerBadges.jsx]: why not just cheese ?  Unless we expect negative values

Comment (blukai) [inline components/Player/PlayerBadges.jsx]: just cheese shows "0"

**Comment by blukai** [comment]
@howardchung, How do you determine whether a pro player?

**Comment by howardchung** [comment]
There's a API endpoint called GetProPlayerList

**Comment by blukai** [comment]
Done 🍀

**Pull request merged**